lib/active_scheduler/resque_wrapper.rb

```diff
@@ -23,7 +25,7 @@ def self.wrap(schedule)
           class:        'ActiveScheduler::ResqueWrapper',
           queue:        queue,
           args: [{
-            job_class:  opts[:class],
+            job_class:  opts[:class] || maybe_class_name,
```

Comment:
Why not just do `opts[:class] || job` here - that way it would work if multiple scheduled jobs are in the .yml file without classnames.